components/tools/OmeroPy/test/unit/clitest/test_web.py

```diff
@@ -175,19 +175,21 @@ def testApacheConfig(self, prefix, cgihost, cgiport, capsys, monkeypatch):
         if prefix:
             missing = self.required_lines_in([
                 ('FastCGIExternalServer ',
-                 'var/omero.fcgi" -host %s -idle-timeout 60' % expected_cgi),
+                 '/var/run/omero.fcgi" -host %s -idle-timeout 60'
+                 % expected_cgi),
                 ('Alias %s/error ' % prefix, 'etc/templates/error'),
                 ('Alias %s ' % static_prefix[:-1],
                  'lib/python/omeroweb/static'),
-                ('Alias %s "' % prefix, 'var/omero.fcgi/"'),
+                ('Alias %s "' % prefix, '/var/run/omero.fcgi/"'),
```

Comment:
Since there's no longer an unknown prefix on `var` you could change this to the non-tuple form:
`'Alias %s "/var/run/omero.fcgi/"' % prefix`

(https://github.com/sbesson/openmicroscopy/blob/apache_stanza_fcgi/components/tools/OmeroPy/test/unit/clitest/test_web.py#L80)

And similarly for the next two changes.